client/my-sites/stats/site.jsx

```diff
@@ -180,6 +182,21 @@ class StatsSite extends Component {
 						period={ this.props.period }
 						chartTab={ this.props.chartTab }
 					/>
+					{ ! isVip && (
+						<Banner
```

Comment:
I think that using the Banner is component is causing an issue where this is appearing for non-admins who shouldn't have access to Earn - this is leading to Calypso crashing for non-admins.

<img width="1671" alt="Screenshot 2020-07-11 at 14 53 57" src="https://user-images.githubusercontent.com/43215253/87225626-696ce680-c386-11ea-95e2-9a26c55f72c8.png">

I'm also not sure this makes sense to show to people who are already earning revenue (the "Start" verb made me think it was a bug when I first saw it!). I've opened up a PR in #44081 to fix these conditions. :)